src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -445,8 +448,30 @@ public int GetIndexesCount()
 
             return count;
         }
+
+        internal SupportedFeatureTypes GetSupportedFeatures()
+        {
+            var supportedFeatures = new SupportedFeatureTypes();
+            if (SupportedFeatures.Contains(nameof(SupportedFeatureTypes.ThrowRevisionKeyTooBigFix)))
+                supportedFeatures.ThrowRevisionKeyTooBigFix = true;
+            
+            return supportedFeatures;
+        }
+
+        internal void SetSupportedFeatures()
+        {
+            if (SupportedFeatures.Contains(nameof(SupportedFeatureTypes.ThrowRevisionKeyTooBigFix)))
+                return;
+
+            SupportedFeatures.Add(nameof(SupportedFeatureTypes.ThrowRevisionKeyTooBigFix));
```

Comment:
Shuldn't that list be kept in e.g Constants? Next to those constants taht you need to add?

Reply:
done